### PR TITLE
✨ Create Header Button

### PR DESCRIPTION
**Pull request by mrana17**
- add in storybook as story
- styled button with image
- https://riding-heroe-headerbutt-q35wix.herokuapp.com/storybook/index.html?path=/story/common-headerbutton--notifications-button

### CODE REVIEW

**Review by DNaunin — APPROVED** [review]
Great Job! 🎉

**Review by sweisz — APPROVED** [review]
GJ 🤖

File: components/headerButton/headerButton.module.css [hunk 1]
```diff
@@ -0,0 +1,17 @@
+.headerButton {
+  display: grid;
+  place-items: center;
+  border: none;
+  border-radius: 0.6rem;
+  background: var(--pattern-color);
+  width: 2.25rem;
+  height: 2.25rem;
+  -webkit-box-shadow: 0 0.2rem 0.4rem 0 #00000029;
+  -moz-box-shadow: 0 0.2rem 0.4rem 0 #00000029;
```
Comment (phlgr) [inline components/headerButton/headerButton.module.css]: These will get auto generated on build time

File: components/headerButton/headerButton.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import styles from "./headerButton.module.css";
+import Link from "next/link";
+
+export type HeaderButtonProps = {
+  onClick: MouseEventHandler<HTMLButtonElement>;
+  imgSrc: string;
+  href: string;
+};
+
+function HeaderButton({ onClick, imgSrc, href, ...props }: HeaderButtonProps) {
```
Comment (phlgr) [inline components/headerButton/headerButton.tsx:11]: Why do you need to accept `...props here`?

**Pull request merged**